samples/java/http/request-response-gradle/README.md

```diff
@@ -1,7 +1,46 @@
-# request-response
+# Java Sample - request-response
 
 This is a sample function for how to create and send HTTP requests via our Java HTTP function.
 
-# Building
+## Prerequisites
```

Comment:
Let's fix the formatting here:
<img width="1162" alt="Screen Shot 2022-09-22 at 12 20 52 PM" src="https://user-images.githubusercontent.com/15917939/191833160-1b3a9094-ed59-4eb3-937b-cb40eab290f9.png">